CHANGELOG.rst

```diff
@@ -2,6 +2,85 @@
 SDK Changelog
 =============
 
+3.1.0 (2025-03-24)
+==================
+
+Feature release
+
+There are additional changes compared to the alpha versions. Continue reading for the full
+changelog, which includes the alpha versions release notes.
+
+.. warning:: SDK 3.1.0 requires the backend API version 3.1.0 or newer.
+
+New features
+------------
+
+* [#4917] Changed placement of form "next", "previous", "continue later" and "log out"
+  buttons.
+* [#445] Adopting typescript in the SDK project.
+    - Added typescript dependency and config files.
+    - Added @open-formulieren/formio-renderer 0.4.0 dependency.
+    - Replaced a number of form field components with components from the new
+      formio-renderer library.
+    - Replaced a number of components and functions to typescript variants.
+    - Added typescript definitions for Form and Submission data.
+    - Upgraded a number of dependencies.
+    - Upgraded to @open-formulieren/design-tokens 0.58.0.
+    - Upgraded to @utrecht/components 7.4.0.
```

Comment:
Not a feature, but project maintenance.

```suggestion
* [#445] Started converting the codebase to Typescript.

    - Upgraded to @utrecht/components 7.4.0 in the process, which may affect (custom) CSS themes.

```

(can you also ensure there's a blank line in the older release notes for nested lists, now there's some bold markup, see https://open-forms.readthedocs.io/en/latest/changelog-sdk.html)